workflows_core/api/api.py

```diff
@@ -198,17 +199,31 @@ def _set_workflow_status(
             )
         if metadata is None:
             metadata = {}
-        return requests.post(
-            url=self._base_url + f"/workflows/{job_id}/status",
-            headers=self._headers,
-            json=dict(
-                metadata=metadata,
-                status=status,
-                workflow_name=workflow_name,
-                additional_information=additional_information,
-                send_email=send_email,
-            ),
-        ).json()
+        if worker_number is None:
```

Comment:
can we not send `worker_numer=None` to the API

Reply:
Nah - this was asked in Linear ticket